pubsub/google/cloud/pubsub/topic.py

```diff
@@ -426,11 +434,16 @@ class Batch(object):
                          before automatically commiting. Defaults to infinity
                          (off).
     :type max_messages: float
+
+    :param max_size: The maximum size that the serialized messages can be
+                     before automatically commiting. Defaults to just under
+                     10 MB.
+    :type max_size: int
     """
     _INFINITY = float('inf')
 
     def __init__(self, topic, client, max_interval=_INFINITY,
-                 max_messages=_INFINITY):
+                 max_messages=_INFINITY, max_size=10e6):
```

Comment:
The [pubsub docs for publishing](https://cloud.google.com/pubsub/docs/publisher#publish) state that the limit is 10MB after encoding (but would seem to imply *per message*).